7.5/hooks/post_push

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+VERSION=$(cat Dockerfile | grep '^ENV VERSION' | cut -d" " -f3)
+TAGS=($VERSION 7)
```

Comment:
One last not obvious thing to do is to remove the `7` from the previous version of this file in the 7.4 folder.  This is a post build docker hook that will also tag the 7.5 image as the 7 image (replace the previous "latest" 7.4.x)